.github/workflows/integration-test.yml

```diff
@@ -15,23 +15,46 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        app-language: [ javascript, python-manual-instrumentation, python-auto-instrumentation, go ]
+        app-language: [ javascript, python-manual-instrumentation, python-auto-instrumentation, go, java-manual-instrumentation, java-auto-instrumentation ]
     env:
       APP_PATH: sample-apps/${{ matrix.app-language }}-sample-app
     steps:
       - name: Checkout This Repo
         uses: actions/checkout@v3
+      - name: Setup Java
+        if: ${{ contains(matrix.app-language, 'java-') }}
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 17
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
           role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_ARN }}
           role-duration-seconds: 1200
           aws-region: us-west-2
-      - name: Run test containers
-        run: docker-compose up --abort-on-container-exit
+      - name: Build with Gradle
+        if: ${{ contains(matrix.app-language, 'java-') }}
+        working-directory: sample-apps/java-sample-app
+        run: |
+          chmod +x gradlew
```

Comment:
do we need to do this? you can change the permission and commit that in the repo.

Reply:
I don't think we need it.  Removed in my latest commit.